connect-file-pulse-filters/src/main/java/io/streamthoughts/kafka/connect/filepulse/config/ExcludeFieldsMatchingPatternsConfig.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2023 StreamThoughts.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.streamthoughts.kafka.connect.filepulse.config;
+
+import java.util.Map;
+import java.util.regex.Pattern;
+import org.apache.kafka.common.config.ConfigDef;
+
+public class ExcludeFieldsMatchingPatternsConfig extends CommonFilterConfig {
```

Comment:
I think the "Patterns" in the name of this class should rather be "Pattern". I believe it's a leftover from a first implementation which supported multiple patterns, but then we removed that logic to keep things simple

Reply:
thanks for the feedback, updated now